behandlingsprosess/src/main/java/no/nav/ung/sak/domene/behandling/steg/kompletthet/VurderKompletthetStegImpl.java

```diff
@@ -91,6 +96,9 @@ public VurderKompletthetStegImpl(EtterlysningRepository etterlysningRepository,
         if (erDigitalBruker) {
             kontrollerInntektEtterlysningOppretter.opprettEtterlysninger(behandlingReferanse);
             programperiodeendringEtterlysningTjeneste.opprettEtterlysningerForProgramperiodeEndring(behandlingReferanse);
+            if (hentInntektHendelserEnabled) {
+                inntektAbonnentTjeneste.opprettAbonnement(behandlingReferanse.getAktørId());
```

Comment:
Bør antagelig ha med periode her, antagelig best med perioder fra årsak for kontroller inntekt